src/user-icon/user-icon.scss

```diff
@@ -116,3 +127,53 @@
     }
   }
 }
+
+@mixin iui-user-icon-list {
+  display: flex;
+
+  .iui-user-icon:not(:first-child) {
+    margin-left: -$iui-xs;
+
+    &.iui-small {
+      margin-left: -$iui-xxs;
+    }
+
+    &.iui-large {
+      margin-left: -6px;
```

Comment:
Why not `$iui-s` (`8px`)?

Reply:
Large is now using `$iui-s` and x-large is now using `$iui-sm`.